docs/newcontent/Community/_index.md

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
As far as I can tell, there is no path from `docs/mkdocs.yml` to `docs/newcontent/<anything>`. Did you mean to include this filesystem branch in this PR?

Reply:
no.